cicecore/cicedynB/general/ice_forcing.F90

```diff
@@ -2025,23 +2033,38 @@ subroutine LY_files (yr)
       endif                     ! master_task
 
       end subroutine LY_files
-      subroutine JRA55_files(yr)
+      subroutine JRA55_gx1_files(yr)
 !
       integer (kind=int_kind), intent(in) :: &
            yr                   ! current forcing year
 
-      character(len=*), parameter :: subname = '(JRA55_files)'
+      character(len=*), parameter :: subname = '(JRA55_gx1_files)'
 
       uwind_file = &
-           trim(atm_data_dir)//'/8XDAILY/JRA55_03hr_forcing_2005.nc'
+           trim(atm_data_dir)//'/8XDAILY/JRA55_gx1_03hr_forcing_2005.nc'
```

Comment:
This line is a problem.  My testing is failing.  Here you have assume the gx1 datasets have new names, but all we've done is add gx3 datasets.  The general plan is to never rename of delete any datasets.  We can make a second copy of the datasets with the gx1 name in them, but then we'll have two copies of that data in our input data area.  We need to be able to support versions of the code before this change and then after.  My recommendation is that we revert to the prior naming convention.